common/src/main/java/de/maxhenkel/voicechat/gui/VoiceChatSettingsScreen.java

```diff
@@ -89,13 +89,15 @@ protected void onUpdate(AudioType type) {
                 }
             }
         });
-        y += 21;
         if (isIngame()) {
             addRenderableWidget(Button.builder(ADJUST_VOLUMES, button -> {
                 minecraft.setScreen(new AdjustVolumesScreen());
-            }).bounds(guiLeft + 10, y, xSize - 20, 20).build());
-            y += 21;
+            }).bounds(guiLeft + xSize / 2 + 1, y, (xSize - 20) / 2 - 1, 20).build());
         }
+        y += 21;
+        addRenderableWidget(new ToggleDisplayNametagsButton(guiLeft + 10, y, (xSize - 20) / 2 - 1, 20));
+        addRenderableWidget(new ToggleOnTalkNametagsButton(guiLeft + xSize / 2 + 1, y, (xSize - 20) / 2 - 1, 20));
+        y += 21;
```

Comment:
This setting should not be in this menu. The cloth config menu would make much more sense.